infra/identity/prod/westeurope/main.tf

```diff
@@ -19,6 +19,56 @@ provider "azurerm" {
   }
 }
 
-data "azurerm_subscription" "current" {}
+resource "azurerm_resource_group" "rg_identity" {
+  name     = "${local.project}-identity-rg"
+  location = local.location
+
+  tags = local.tags
+}
+
+module "federated_identities" {
+  source = "../../../modules/azure_identity_federation_github"
```

Comment:
is this module supposed to be put into dx repo later?

Reply:
Yes, dx repo will have an exact copy of this code (module + internal adoption)